schema/application/resolvers/Checklist.ts

```diff
@@ -122,23 +223,140 @@ export const Checklist: ChecklistResolvers = {
     return (await ctx.orm.description.load(parent.id)) as Description;
   },
   async items(parent, args) {
-    // NOTE: We don't support pagination of ChecklistResults yet, even though
-    // the API suggests otherwise...
-    const { first, last } = args;
-    const { id, type } = decodeGlobalId(parent.id);
+    const { type: parentType, id: parentId } = decodeGlobalId(parent.id);
 
-    const rows = await match(type)
+    const { cursor, direction, limit } = buildPaginationArgs(args, {
+      defaultLimit: Number(
+        process.env.DEFAULT_ATTACHMENT_PAGINATION_LIMIT ?? 20,
+      ),
+      maxLimit: Number(process.env.MAX_ATTACHMENT_PAGINATION_LIMIT ?? 20),
+    });
+
+    // Our (default) order clause specifies:
+    // 1. workresultorder ASC
+    // 2. workresultid ASC
+    const cmp = direction === "forward" ? sql`>` : sql`<`;
```

Comment:
so we are sorting from the API now?

Reply:
Always have been. This just makes it paginated.